packages/connectivity/src/scp-cf/destination/destination-cache.ts

```diff
@@ -140,16 +140,23 @@ function cacheRetrievedDestination(
   const expirationTime = expiresIn
     ? Date.now() + parseInt(expiresIn) * 1000
     : undefined;
-  cache.set(key, destination, expirationTime);
+  cache.set(key, { entry: destination, expires: expirationTime });
 }
 
 /**
- * @internal
+ * Set the destination cache instance.
```

Comment:
[nth] Extend the JS doc and explain we have a default but if you want to use some different use this method before the first `getDestination` eitther direct or via `executeHttpRequest` and so on.